zebra/zapi_msg.c

```diff
@@ -2243,6 +2243,12 @@ static void zread_vrf_label(ZAPI_HANDLER_ARGS)
 	s = msg;
 	STREAM_GETL(s, nlabel);
 	STREAM_GETC(s, afi);
+
+	if (!(IS_VALID_AFI(afi))) {
+		zlog_warn("Invalid AFI for VRF label: %s", family2str(afi));
```

Comment:
doesn't seem like `family2str()` will output anything very useful here - could we have the actual afi value too or instead?

Reply:
It prints "?", so sure, the number isp robably better